action_tutorials/action_tutorials_cpp/include/action_tutorials_cpp/visibility_control.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ACTION_TUTORIALS_CPP__VISIBILITY_CONTROL_H_
```

Comment:
C++ headers should have the extension `.hpp`.

Reply:
This file contains C code only.